lib/UserSearch/styles.scss

```diff
@@ -64,27 +64,32 @@
   }
 }
 
-.user-search__name {
-  display: flex;
-  align-items: center;
-  .icon {
-    margin-left: $layout-spacing-base/4;
-    svg {
-      width: $layout-spacing-base/2;
-    }
-  }
-}
-
-.user-search__user {
-  border-top: 1px solid $neutral-light;
-  &:last-child {
-    border-bottom: 1px solid $neutral-light;
-  }
-}
-
 .user-search__text {
   font-size: $typo-size-small;
   color: $neutral-base;
   font-style: italic;
   padding-left: $layout-spacing-base/4;
 }
+
+.user-search-list {
+  .user-search-list__user {
+    border-top: 1px solid $neutral-light;
+    list-style: none;
+
+    &:last-child {
+      border-bottom: 1px solid $neutral-light;
+    }
+
+    .user-search-list__user-name {
+      display: flex;
+      align-items: center;
+
+      .icon {
+        margin-left: $layout-spacing-base/4;
```

Comment:
I've been handling icon spacing with helpers. It removes the need for these little bits of `scss` every time we add icons to any component 😄 

What are your thoughts are using a helper here instead of this?